resources/features/Sic.feature

```diff
@@ -0,0 +1,272 @@
+Feature: A sic calculation groups the input data by substrings of SIC and ID to determine which subgroup contains the highest amount of employees
+         and assigns a subdivision determined by the division and group.
+
+         The method uses the following substrings:
+         first 2 characters as division,
+         first 3 characters as group,
+         first 4 characters as class,
+         and the whole whole SIC.
+
+         Special cases decide subdivision:
+         First if group equals 461, 478 or 479 then the subdivision is A
+         Then if division equals 46 or 47 the subdivision is B
+         Subdivision is C otherwise
+
+    @Happy Path
+    Scenario Outline: Happy Path - Origional Happy Path
+        Given input:
```

Comment:
It would help from a readability point of view if the input files was ordered by ern, that way you could visually check against the output easily